app.py

```diff
@@ -1,19 +1,2 @@
-from flask import Flask, render_template
-
-
-app = Flask(__name__)
-
-
-@app.route('/')
-def home():
-    return render_template('error.html')
-
-
-@app.route('/<string:token>')
-def survey(token):
-    return render_template('survey.html')
-
-
-@app.route('/end', methods=['post'])
-def end():
-    return render_template('end.html')
+from core import app
+from core.views import *
```

Comment:
When importing `app`, because it is [in the module that connects  to the database](https://github.com/anthropedia/tci-online/pull/9/files#diff-3464e388711711d3f415d392db0de2ceR10), the connection should be done implicitely, indeed.
If it works fine it should be good to ship :)